ts/parts/Chart.ts

```diff
@@ -1051,119 +1059,121 @@ extend(Chart.prototype, /** @lends Highcharts.Chart.prototype */ {
+            color: '${palette.neutralColor80}',
+            fontSize: this.options.isStock ? '16px' : '18px' // #2944
+        } : {
+            color: '${palette.neutralColor60}'
+        };
+
+        // Merge default options with explicit options
+        const options = this.options[name] = merge(
             // Default styles
-            (!styledMode && {
-                style: {
-                    color: '${palette.neutralColor60}'
-                }
-            }) as Highcharts.SubtitleOptions,
-            options.subtitle as Highcharts.SubtitleOptions,
-            subtitleOptions as Highcharts.SubtitleOptions
+            (!this.styledMode && { style }) as
+                Highcharts.DescriptionOptionsType,
+            this.options[name],
+            explicitOptions
         );
 
+        let elem = this[name];
 
-        // add title and subtitle
-
-        /**
-         * The chart title. The title has an `update` method that allows
-         * modifying the options directly or indirectly via
-         * `chart.update`.
-         *
-         * @sample highcharts/members/title-update/
-         *         Updating titles
-         *
-         * @name Highcharts.Chart#title
-         * @type {Highcharts.TitleObject}
-         */
+        if (elem && explicitOptions) {
+            this[name] = elem = (elem as any).destroy(); // remove old
+        }
 
-        /**
-         * The chart subtitle. The subtitle has an `update` method that
-         * allows modifying the options directly or indirectly via
-         * `chart.update`.
-         *
-         * @name Highcharts.Chart#subtitle
-         * @type {Highcharts.SubtitleObject}
-         */
+        if (options && !elem) {
+            elem = this.renderer.text(
+                options.text as any,
+                0,
+                0,
+                options.useHTML
+            )
+                .attr({
+                    align: options.align,
+                    'class': 'highcharts-' + name,
+                    zIndex: (options as any).zIndex || 4
+                })
+                .add();
 
-        [
-            ['title', titleOptions, chartTitleOptions],
-            ['subtitle', subtitleOptions, chartSubtitleOptions]
-        ].forEach(function (
-            arr: Array<(
-                string|
-                Highcharts.TitleOptions|
-                Highcharts.SubtitleOptions|
-                undefined
-            )>,
-            i: number
-        ): void {
-            var name = arr[0] as string,
-                title = (chart as any)[name],
-                titleOptions = arr[1],
-                chartTitleOptions = arr[2] as (
-                    Highcharts.TitleOptions|Highcharts.SubtitleOptions
-                );
+            // Update methods, shortcut to Chart.setTitle, Chart.setSubtitle and
+            // Chart.setCaption
+            elem.update = function (
+                o: (Highcharts.DescriptionOptionsType)
```

Comment:
I suggest a more descriptive variable like (at least) `opt`. Compiler will reduce it anyway.